jQuery/honeyHeadquarters.js

```diff
@@ -1,25 +1,25 @@
 // $.ajax('https://sheetsu.com/apis/v1.0/64b5c3f8').done(function(data) {
 $.ajax('http://localhost:3000/citizens').done(function(citizens) {
```

Comment:
So as far as everything goes in this `$.ajax` function, you're doing a lot. In fact, you're defining all of your functions inside of the callback function. 

Perfectly valid, but what happens when the callback executes? what if you want to make a call to another endpoint of some other api and you want to reuse these functions?

yeah that's right. you can't! D:

let's repurpose this file a little bit and think about structure:

let's call this file something that denotes that it's the prime mover, e.g. `index.js` or `main.js` or `driver.js`
Secondarily, let's move your ajax block into another file called `ajax.js`. This gives a cleaner separation of concerns, as we know we need to do two things in the driver file:

1: use ajax to get our stuff
2: use jQuery to populate our DOM.